Content.Shared/Chemistry/Components/Solution.Managerial.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Shared.Chemistry.Reagent;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Shared.Chemistry.Components
+{
+    public partial class Solution
+    {
+
+        /// <summary>
+        ///     If reactions will be checked for when adding reagents to the container.
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("canReact")]
+        public bool CanReact { get; set; } = true;
+
+        /// <summary>
+        ///     Volume needed to fill this container.
+        /// </summary>
+        [ViewVariables]
+        public ReagentUnit AvailableVolume => MaxVolume - CurrentVolume;
+        
+        public ReagentUnit DrawAvailable => CurrentVolume;
+        public ReagentUnit DrainAvailable => CurrentVolume;
+
+        /// <summary>
+        ///     Checks if a solution can fit into the container.
+        /// </summary>
+        /// <param name="solution">The solution that is trying to be added.</param>
+        /// <returns>If the solution can be fully added.</returns>
+        public bool CanAddSolution(Solution solution)
+        {
+            return solution.TotalVolume <= AvailableVolume;
+        }
+
+        [DataField("maxSpillRefill")]
+        public ReagentUnit MaxSpillRefill { get; set; }
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("maxVol")]
+        public ReagentUnit MaxVolume { get; set; } = ReagentUnit.Zero;
+
+        [ViewVariables]
+        public ReagentUnit CurrentVolume => TotalVolume;
+
+        [ViewVariables]
+        public EntityUid OwnerUid { get; set; }
```

Comment:
Looking at the code I don't see any instances of this where it can't be replaced by a better alternative? I looked at 5 spots and all of them already have access to the solution owner's entityuid already. Please leave a TODO to refactor this out as I really don't think we need it.

Splitting and cloning doesn't even copy it across as well.